includes/data-stores/class-wc-admin-reports-downloads-stats-data-store.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/**
+ * WC_Admin_Reports_Downloads_Data_Store class file.
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Reports_Downloads_Data_Store.
+ */
+class WC_Admin_Reports_Downloads_Stats_Data_Store extends WC_Admin_Reports_Downloads_Data_Store implements WC_Admin_Reports_Data_Store_Interface {
+
+	/**
+	 * SQL columns to select in the db query and their mapping to SQL code.
+	 *
+	 * @var array
+	 */
+	protected $report_columns = array(
+		'download_count' => 'COUNT(DISTINCT download_log_id) as download_count',
```

Comment:
I'm also seeing a "days" stat in the designs.  I’m not really sure what this is referring to or if it has been removed, but curious if it needs to be added here.  /cc @LevinMedia

Reply:
@joshuatf @justinshreve  Days should be listed at the bottom of the tabular data, and count the number of days selected in the date range picker. 

Outside the scope of this pr, but the report is missing the date range picker.